apm-lambda-extension/main.go

```diff
@@ -60,12 +60,17 @@ func main() {
 	// pulls ELASTIC_ env variable into globals for easy access
 	config := extension.ProcessEnv()
 
-	// setup http server to receive data from agent
-	// and get a channel to listen for that data
+	// Create a channel to buffer apm agent data
 	agentDataChannel := make(chan extension.AgentData, 100)
 
+	// Start http server to receive data from agent
 	extension.StartHttpServer(agentDataChannel, config)
 
+	// Create a client to use for sending data to the apm server
+	client := &http.Client{
+		Transport: http.DefaultTransport.(*http.Transport).Clone(),
+	}
+
 	// Make channel for collecting logs and create a HTTP server to listen for them
 	logsChannel := make(chan logsapi.LogEvent)
```

Comment:
The PR description mentions _removed all the logsapi code and tests, as we won't need them anymore with these changes._ -- is the PR description out of date, or does this code not belong here? (or some third thing)

Reply:
Sorry the description is out of date! I'll update it.